src/arraytraits.rs

```diff
@@ -182,6 +183,7 @@ where
     /// let array = Array::from_iter((0..5).map(|x| x * x));
     /// assert!(array == arr1(&[0, 1, 4, 9, 16]))
     /// ```
+    #[track_caller]
```

Comment:
Should not have track caller - it's a trait that doesn't support it, and the panic reason is very obscure, not worth it.